.github/workflows/chromaticqa.yml

```diff
@@ -28,4 +24,6 @@ jobs:
       run: yarn chromatic --exit-zero-on-changes
       env:
         CI: true
+        CHROMATIC_BRANCH: ${{ github.event.pull_request.head.ref }}
```

Comment:
The Chromatic team just added these yesterday. We're able to explicitly set the branch name as a baseline